shiny/ui/_input_select.py

```diff
@@ -192,6 +195,55 @@ def input_select(
     * :func:`~shiny.ui.input_radio_buttons`
     * :func:`~shiny.ui.input_checkbox_group`
     """
+    if selectize:
+        warn_deprecated(
+            "`selectize` parameter of `input_select()` is deprecated. "
+            "Use `input_selectize()` instead of passing `selectize=True`."
+        )
+
+    if remove_button is not None:
+        warn_deprecated(
+            "`remove_button` parameter of `input_select()` is deprecated. "
+            "Use `input_selectize()` instead."
+        )
+
+    if options is not None:
+        warn_deprecated(
+            "`options` parameter of `input_select()` is deprecated. "
+            "Use `input_selectize()` instead."
+        )
```

Comment:
For all three parameters, please change the parameter type to `MISSING_TYPE` and default the value to `DEPRECATED` ([link](https://github.com/posit-dev/py-shiny/blob/785197861d209e04a2498972143660aaa10f6a96/shiny/types.py#L48))

This will clearly signal that it should not be used. (Similar to how `lifecycle::deprecated()` is used in R packages)

To check if the value is present, use code similar to `if isinstance(selectize, MISSING_TYPE):` for the three cases.

Thank you